integration-tests/flake.nix

```diff
@@ -0,0 +1,92 @@
+{
+  description = "Integration tests";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+
+    # define gomod2nix which is a dependency to build Go lang plugins
+    gomod2nix = {
+      url = "github:tweag/gomod2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+      inputs.utils.follows = "flake-utils";
+    };
+
+    # define chainlink-dev flake as a input
+    chainlink-dev.url = "git+ssh://git@github.com/smartcontractkit/chainlink-project-nix-poc?ref=main&dir=chainlink-dev"; # this could also be a project URL
+    chainlink.url = "git+ssh://git@github.com/smartcontractkit/chainlink-project-nix-poc?ref=main&dir=chainlink";
```

Comment:
This is actually the work we will need to move to this core repo flake.nix, as here we build the chainlink core node as Nix package and the core repo should expose that - @willyrgf do we have a ticket to track this? cc @andrevmatos